Framework/CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ set(EXE_SRCS
     src/runDataProducer.cxx
     src/runQC.cxx
     src/runBasic.cxx
+    src/runTPCQCPID.cxx
```

Comment:
Indeed this is what we would like to have eventually, to be independent of the input type. Since there is already a generic tree reader in o2 I asked @matthiasrichter to implement a generic configurable workflow for this.